apps/meteor/app/models/server/models/LivechatVisitors.js

```diff
@@ -247,6 +247,20 @@ export class LivechatVisitors extends Base {
 		const query = { _id };
 		return this.remove(query);
 	}
+
+	removeContactManagerByUsername(manager) {
+		const query = {
+			contactManager: {
+				username: manager,
```

Comment:
Since this is a query, do you mind adding an index on this property?

Reply:
Makes sense, adding!